src/sentry/utils/pytest/sentry.py

```diff
@@ -159,6 +159,19 @@ def pytest_configure(config):
     from sentry import http
     http.DISALLOWED_IPS = set()
 
+    from sentry.utils.db import is_postgres
```

Comment:
this code should probably happen after ``fix_south`` is called.

Reply:
It does. Unless you mean immediately after. But either way, I'm gonna see what makes things happy here. This was me just guessing.

Reply:
I mean immediately after, but not sure if it will actually cause issues by keeping it below. Theres sort of a "things should be in an ok state at this point", and I'd just suggest bundling the "database should be ready" parts.